MOCC/src/configurations/getdevicemeta.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+import PyTango
+import json
+
+def get_device_meta(self,device_id):
```

Comment:
if its part of a particular device server, wouldn't it be applicable to only that device?
Could you tell about this a bit more